lambda/service/compute_trigger/compute_trigger.go

```diff
@@ -21,41 +21,41 @@ type Trigger interface {
 }
 
 type ComputeTrigger struct {
-	Client                      clients.Client
-	Integration                 models.WorkflowInstance
-	Store                       store_dynamodb.DynamoDBStore
-	WorkflowInstanceStatusStore store_dynamodb.WorkflowInstanceStatusDBStore
-	OrganizationId              string
+	Client                               clients.Client
+	Integration                          models.WorkflowInstance
+	Store                                store_dynamodb.DynamoDBStore
+	WorkflowInstanceProcessorStatusStore store_dynamodb.WorkflowInstanceProcessorStatusDBStore
+	OrganizationId                       string
 }
 
 func NewComputeTrigger(
 	client clients.Client,
 	integration models.WorkflowInstance,
 	store store_dynamodb.DynamoDBStore,
-	workflowInstanceStatusStore store_dynamodb.WorkflowInstanceStatusDBStore,
+	workflowInstanceProcessorStatusStore store_dynamodb.WorkflowInstanceProcessorStatusDBStore,
 	organizationId string,
 ) Trigger {
-	return &ComputeTrigger{client, integration, store, workflowInstanceStatusStore, organizationId}
+	return &ComputeTrigger{client, integration, store, workflowInstanceProcessorStatusStore, organizationId}
 }
 
 // runs trigger
 func (t *ComputeTrigger) Run(ctx context.Context) error {
 	id := uuid.New()
 	integrationId := id.String()
-	startedAt := time.Now().UTC()
+	now := time.Now().UTC()
 
 	// persist to dynamodb
 	store_integration := store_dynamodb.WorkflowInstance{
 		Uuid:                  integrationId,
-		Name:                  utils.RunName(t.Integration.Name, startedAt),
+		Name:                  utils.RunName(t.Integration.Name, now),
 		ComputeNodeUuid:       t.Integration.ComputeNode.ComputeNodeUuid,
 		ComputeNodeGatewayUrl: t.Integration.ComputeNode.ComputeNodeGatewayUrl,
 		DatasetNodeId:         t.Integration.DatasetNodeID,
 		PackageIds:            t.Integration.PackageIDs,
 		Workflow:              t.Integration.Workflow,
 		Params:                t.Integration.Params,
 		OrganizationId:        t.OrganizationId,
-		StartedAt:             startedAt.String(),
```

Comment:
this is an out of band functional change but... the workflow isn't really started when its created.. its started when it gets picked up by the workflow manager